client/webserver/site/src/js/charts.ts

```diff
@@ -1018,7 +1018,7 @@ export class CandleChart extends Chart {
     if (!data.candles) return
     this.market = market
     const [qFactor, bFactor] = [quoteUnitInfo.conventional.conversionFactor, baseUnitInfo.conventional.conversionFactor]
-    this.rateConversionFactor = RateEncodingFactor * bFactor / qFactor
+    this.rateConversionFactor = RateEncodingFactor * qFactor / bFactor
```

Comment:
Does `bconv` in `setHighLow` also need to change or is that inheriting this?

Reply:
The USDC market is a good illustration of the shifts:

![image](https://user-images.githubusercontent.com/9373513/217343154-37e59f2e-becf-45eb-ba14-eb655954e295.png)

Looks like in the case of `bconv` it's neglecting the `RateEncodingFactor`

Reply:
It was using `bconv`, but it should be `qconv`.